tex2xml.sh

```diff
@@ -22,8 +22,10 @@ tex = re.sub(r"(label{)(.*?)(})", lambda m: m[0].replace(" ", r"_"), tex)
 with open("$1_copy.tex", "w", encoding='utf-8') as fi:
     fi.write(tex)
 END
+# Sort the bib file alphabetically by first author
+python3 sort_bib.py $2
 # Convert tex file to jats xml file
-pandoc $1_copy.tex -f latex -t jats+element_citations --citeproc --bibliography=$2.bib --mathjax --metadata link-citations=true --natbib --csl apa.csl -s -o $1.xml
+pandoc $1_copy.tex -f latex -t jats+element_citations --citeproc --bibliography=$2_sort.bib --mathjax --metadata link-citations=true --natbib --csl apa.csl -s -o $1.xml
```

Comment:
suggest changing the following to prevent the command from misinterpreting the variable names:
- `$1_copy.tex` to `${1}_copy.tex`
- `$2_sort.bib` to `${2}_sort.bib` 

Also applies to lines 30 and 31.